src/ConsoleApp/consoleAppDemo.java

```diff
@@ -7,14 +7,40 @@
  * @author Louis
  *
  */
-public class consoleAppDemo {
 
+import java.util.Scanner;
+
+
+
+public class consoleAppDemo {
+	
 	/**
 	 * @param args
 	 */
+	
+	
+	
+	
 	public static void main(String[] args) {
+
+		
+		System.out.println("Brewmaster");
+
+		System.out.println("press any key to continue");
+		
+		
+		Scanner in = new Scanner(System.in);
+		in.next();
+		System.out.println("End");
```

Comment:
I have researched  it and we will change the requirement to "press enter to continue"

Reply:
Remove this line that prints 'end'. It is not not part of the Acceptance Criteria.